pkg/controller/spire-oidc-discovery-provider/configmaps.go

```diff
@@ -28,6 +30,8 @@ func GenerateOIDCConfigMapFromCR(cr *v1alpha1.SpireOIDCDiscoveryProvider) (*core
 	jwtIssuer := cr.Spec.JwtIssuer
```

Comment:
nit: rename cr as it is generic, add a better identifier for readability. Maybe "dp".